client/core/src/com/cyberbot/bomberman/controllers/ScreenChangeInterface.java

```diff
@@ -0,0 +1,8 @@
+package com.cyberbot.bomberman.controllers;
+
+import com.cyberbot.bomberman.screens.ScreenState;
+
+public interface ScreenChangeInterface {
+    void setScreen(ScreenState state);
+    public void setScreen(ScreenState state, String playerName, boolean isOwner);
```

Comment:
Implement separate methods for each `ScreenState`, this allows for better metadata handling

Reply:
fixed